src/test/mocks/store.ts

```diff
@@ -112,4 +114,8 @@ export const mockStore: ReduxState = {
     byValidator: {},
     validatorIds: [],
   },
+  foxEth: {
+    farmOpportunities: [],
+    lpOpportunity: {} as EarnOpportunityType,
+  },
```

Comment:
I know this is early work on the notion of data for DeFi opportunities, and is subject to change, but we should probably namespace this.

What are we solving, and what is this supposed to represent?
IMO, something as simple as `defi` would make sense to start with. We might need to represent other data with this, but in the near future, this is effectively going to bring DeFi data into state in a normalized fashion